src/codewhisperer/util/telemetryHelper.ts

```diff
@@ -417,6 +412,10 @@ export class TelemetryHelper {
         return i === acceptIndex ? 'Accept' : 'Ignore'
     }
 
+    public getCompletionType(i: number, completionTypes: Map<number, CodewhispererCompletionType>) {
```

Comment:
don't think we need another method for its purpose?

Reply:
It's just a wrapper to hide the `||` logic so caller don't have to worry about it.